### ISSUE TITLE
setting server.timeout

### ISSUE THREAD

**Issue by ttghr** [Lesterpig/sockpress#10]
Hi,
the sockpress server has no oppertunity to set server.timeout.
A new property `serverTimeout` in the options solve this problem.
In index.js:
```javascript
if (options.serverTimeout && options.serverTimeout>=0)
  _server.setTimeout(options.serverTimeout);
```

**Comment by Lesterpig** [comment]
Hi!

Thank you for pointing out this issue. I'll check this as soon as possible.
Tests are broken under the last version of NPM and I have to do some refactor before adding anything to this module.

**Comment by Lesterpig** [comment]
Ok, I've probably a better idea regarding this issue.

What if Sockpress populates the express application with the raw `http` or `https` server? You could then customize this server with any option you want with something like `app.rawServer.setTimeout(...)`. Express provides maybe a way to access the server, but I've not found anything yet...

**Comment by ttghr** [comment]
You are right. That is more general and prevents other configuration issues.

**Comment by Lesterpig** [comment]
Added and available in sockpress@1.0.2 :+1:
